app/controllers/api/v1/trello_boards_controller.rb

```diff
@@ -0,0 +1,23 @@
+require 'trello'
```

Comment:
do you need this for `Trello Service`? maybe it makes sense to put this in `Trello service` file?